lang/en/website-notice.yaml

```diff
@@ -0,0 +1,12 @@
+banner:
+  title: Bitcoincash.org and its owners no longer accurately represent Bitcoin Cash!
+  sub: Click to find out more
+
+title: Notice to services that use Bitcoin Cash (BCH)
+h1: <span class="text-muted">Bitcoincash.org</span> is unreliable for information regarding the BCH network
```

Comment:
`information regarding the Bitcoin Cash (BCH) network`

I recommend more formal titles.